recipes/fastlinkingnumbers/build.sh

```diff
@@ -0,0 +1,14 @@
+
+mkdir build
+cd build
+cmake ..
+make
+# Copy executables
+mkdir -p $PREFIX/bin
+cp verifycurves $PREFIX/bin/
+cp obj2bcc $PREFIX/bin/
+# Copy shared library and headers
+mkdir -p $PREFIX/lib
+mkdir -p $PREFIX/include
+cp -P libverifycurves* $PREFIX/lib
+cp ../include/* $PREFIX/include
```

Comment:
Is there no `make install` target?

Reply:
Should the headers be installed directly into `include/` without any prefix for the library? (Maybe, I am not sure what upstream is doing here.)

If so, maybe that should be pointed out as a problem to upstream. Headers `model.h` and `curve.h` are extremely generic and this will likely lead to conflicts.

Reply:
Ok. Maybe you could make your life easier by patching an install target into the CMakeLists file (and upstream the patch.) But either way works then.

Since upstream does not install the headers at all yet, one might even consider not to install them at all. It seems that this library is currently not meant to be consumed by any other package but just exist so the binaries work?

In any case, I would try to ask upstream about the installation location first. It's always annoying if headers are installed in different directories in different distributions as this might then lead to:
```
#ifdef CONDA_FORGE
#include something
#else
#include something else
#endif
```